apps/daimo-mobile/src/view/screen/HomeScreen.tsx

```diff
@@ -44,6 +53,14 @@ function HomeScreenInner({ account }: { account: Account }) {
   const translationY = useSharedValue(0);
   const [isActionVisible, setIsActionVisible] = useState(false);
 
+  const historyStatus = useRef<CountStatus>({});
+  const [histListMini, setHistListMini] = useState(
+    <HistoryListSwipe account={account} showDate={false} maxToShow={5} />
+  );
+  const [histListFull, setHistListFull] = useState(
+    <HistoryListSwipe account={account} showDate />
+  );
```

Comment:
ive never seen this pattern of `useState` with a rendered component...

the CountStatus trick is elegant & makes sense, but maybe we can prevent HistListSwipe rerendering a different way?

eg out in HomeScreen,

```
const [account] = useAccount();
if (!account) return null;
const countStatus = countBy(...);
const accountMemo = useMemo(() => account, [countOpStatus]);
```
...then accountMemo stays reference equal until the status changes